imports/api/users/profiles_collection.js

```diff
@@ -18,4 +18,14 @@ export const Profiles = new scorpius.collection('profiles', {
   }
 });
 
+Profiles.allow({
+  insert: function (userId, doc, fieldNames, modifier) {
+    return Profiles.findOne({userId: userId}) !== undefined;
+  },
+  update: function (userId, doc, fieldNames, modifier) {
+    return userId === doc.userId || Profile.findOne({userId: userId}).keys.length === 1;
+  }
```

Comment:
So, questions about this: 

- Does this affect how these Profiles will be managed in Scorpius? We want only Admins to be able to modify other people's Profiles (thru the Scorpius dash). Otherwise, people should just be able to update their own (via the profile_edit page).
- Do we need to allow insert? Since we're creating the profile automatically and just allowing ppl to update their own profiles, I wonder if it's needed.